### ISSUE TITLE
Training Doesn't Stop: Prolonged Training Time on 3090

### ISSUE THREAD

**Issue by cyrcule** [chroneus/textual_avatar#1]
I've been running a training session on an NVIDIA 3090, and after more than 9 hours, the training still hasn't stopped. Here are the metrics from the latest checkpoint:
{
    'loss': 0.356,
    'learning_rate': 0.001,
    'epoch': 0.09
}
Can you please guide me on why the training might be taking so long and any potential solutions? Any help would be appreciated.

**Comment by cyrcule** [comment]
wandb project [link](https://wandb.ai/false_team/huggingface/runs/85uct2o8/overview)

**Comment by chroneus** [comment]
Thanks for evaluating my project. 

The training time is depend on the number of dialogues, in my case it was about 17k lines, so it takes about 8 hours each epoch. The loss 0.356 produces somehow meaningful reply for me, but I was enjoyed for less than 0.3

Due to bug in SFTTrainer the epoch ticker is not correct, I remember that after 0.1 I've got next epoch. This behavior was due to "packing" optimization in learning.

P.S. please hide wandb logs, they may disclose your dialogs during evaluation.
